src/components/add-tweet.tsx

```diff
@@ -52,7 +56,11 @@ const AddTweet = () => {
         style={{ borderRadius: "0 0 0.75rem 0.75rem" }}
       >
         <Text>Tell the world what's in your mind!</Text>
-        <Button handleClick={handleSubmit} value="Tweet" />
+        <Button
+          value="Tweet"
+          handleClick={handleSubmit}
+          disabled={buttonState}
```

Comment:
you can directly give the condition here 
message.length or message.length === 0 ? true :false